code/modules/mob/living/carbon/diona_base.dm

```diff
@@ -50,7 +50,7 @@ var/list/diona_banned_languages = list(
 	DS.last_location = loc
 	if(light_amount)
 		adjustNutritionLoss(-light_amount) // regenerate our nutrition from light.
-	move_delay_mod = min(initial(move_delay_mod) - light_amount * 1.5, 0)
+	move_delay_mod = min(initial(move_delay_mod) - light_amount * 1, 0)
```

Comment:
This change actually speeds up diona from light

Reply:
Yes, I did test it.

Reply:
Yeah, yours is right. I forgot it is going negative numbers to make it go faster.